src/external/rawspeed/data/cameras.xml

```diff
@@ -5734,6 +5734,40 @@
 		<Crop x="0" y="0" width="-74" height="0"/>
 		<Sensor black="15" white="3971"/>
 	</Camera>
+	<Camera make="Panasonic" model="DMC-LX15">
+		<ID make="Panasonic" model="DMC-LX15">Panasonic DMC-LX15</ID>
+		<CFA width="2" height="2">
+			<Color x="0" y="0">GREEN</Color>
+			<Color x="1" y="0">BLUE</Color>
+			<Color x="0" y="1">RED</Color>
+			<Color x="1" y="1">GREEN</Color>
+		</CFA>
+		<Crop x="0" y="0" width="0" height="0"/>
+		<Sensor black="142" white="4095" iso_list="80 100 125 400"/>
```

Comment:
Something is wrong here, i get different results, especially for ISO80, especially for ISO80 whitelevel.

```
../tools/dngmeta.rb lx15/*.DNG
invsensors {[142, 2095]=>[80], [142, 3400]=>[100], [142, 4095]=>[125], [143, 4095]=>[160, 200, 250, 320, 400], [144, 4095]=>[640, 800], [145, 4095]=>[1250, 1600], [149, 4095]=>[3200], [154, 4095]=>[6400], [166, 4095]=>[12800], [167, 4095]=>[25600]}

sensors [[80, [142, 2095]], [100, [142, 3400]], [125, [142, 4095]], [160, [143, 4095]], [200, [143, 4095]], [250, [143, 4095]], [320, [143, 4095]], [400, [143, 4095]], [640, [144, 4095]], [800, [144, 4095]], [1250, [145, 4095]], [1600, [145, 4095]], [3200, [149, 4095]], [6400, [154, 4095]], [12800, [166, 4095]], [25600, [167, 4095]]]
mostfrequent [[143, 4095], [160, 200, 250, 320, 400]]
invsensors {[142, 2095]=>[80], [142, 3400]=>[100], [142, 4095]=>[125], [144, 4095]=>[640, 800], [145, 4095]=>[1250, 1600], [149, 4095]=>[3200], [154, 4095]=>[6400], [166, 4095]=>[12800], [167, 4095]=>[25600]}

        <Camera make="Panasonic" model="DMC-LX15">
                <ID make="" model="">Panasonic DMC-LX9</ID>
                <Sensor black="143" white="4095"/>
                <Sensor black="142" white="2095" iso_list="80"/>
                <Sensor black="142" white="3400" iso_list="100"/>
                <Sensor black="142" white="4095" iso_list="125"/>
                <Sensor black="144" white="4095" iso_list="640 800"/>
                <Sensor black="145" white="4095" iso_list="1250 1600"/>
                <Sensor black="149" white="4095" iso_list="3200"/>
                <Sensor black="154" white="4095" iso_list="6400"/>
                <Sensor black="166" white="4095" iso_list="12800"/>
                <Sensor black="167" white="4095" iso_list="25600"/>
        </Camera>
```